src/test/test.iml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file

Reply:
done

Reply:
Please remove src/main/main.iml as well.